### PR TITLE
Adding updated metrics + formatting

### PR DESCRIPTION
**Pull request by adiramani**
### Changes
- Updating the `single_file_postprocessing` to accept multiple threshold probability values, as per request from Dierdre. 
- Updating the constants to allow for aggregation across all of the default thresholds
- Updating `country_lvl_aggregate` to calculate the number of ius that reach a threshold for all the given "threshold metrics".

### Testing
Ran it on local oncho data to verify the output is as expected.
Updated tests to account for parameter changes, and validated that the test suite passed.

### CODE REVIEW

File: tests/post_processing/test_single_file_post_processing.py [hunk 1]
```diff
@@ -132,6 +132,40 @@ def test_calculate_probabilities_and_thresholds_success():
...
+            measure_column_loc=test_input["measure_loc"],
+            age_start_column_loc=test_input["age_start_loc"],
+            age_end_column_loc=test_input["age_end_loc"],
+            draws_loc=test_input["draws_loc"],
+            prevalence_marker_name=PREV_MEASURE_NAME,
+            threshold=0.1,
+            pct_runs_under_threshold=[0.8, 0.9],
+        )
+    )
+    assert (
```
Comment (thk123) [inline tests/post_processing/test_single_file_post_processing.py:150]: ❓ I'm struggling to follow this test to be sure it is correct. In particular I can't see how we know it has indeed worked out the right values for 0.8 and 0.9.

**Review by thk123 — COMMENTED** [review]
Tests I'd expect to see:

- [ ] Providing a single float for the confidence ranges and have it transformed into a list (or just drop the backwards compatibility bit 😉 )
- [x] Two confidence requirements provided, two corresponding columns with correct names and values - partially done by `test_calculate_probabilities_and_thresholds_success` I believe. 
- [x] Updating a aggregate country for new column names looks to be done by `test_country_lvl_aggregate_aggregate_by_country_rename` though I can't say I fully understand this test 
- [ ] country counts of IUs meeting threshold measure


General comment - it is easier to review things if formatting changes are in separate commits from actual changes (e.g. the splitting of `ONCHO_MEASURES` into multiple lines should not be in a commit that also includes functional changes)

**Comment by thk123** [comment]
I think I'm a bit confused by the country level aggregation, and the test didn't really clarify it for me tbh - perhaps we can have a chat tomorrow to work through this.

File: endgame_postprocessing/post_processing/single_file_post_processing.py [hunk 2]
```diff
@@ -90,11 +99,15 @@ def _calculate_probabilities_and_thresholds(
                                         compare with the threhsold.
         threshold (float): the value of the threshold we compare prevalence values
                             to (value < threhsold).
-        pct_runs_under_threshold (float): the percent of runs that should reach the given threshold
+        pct_runs_under_threshold (list[float]): the percentages of runs that should reach the given threshold
                                             (pct_runs >= pct_runs_under_threshold).
     Returns:
         Returns a 2D Matrix with the post-processed metrics for the given input
     """
+    # for backwards compatibility
```
[the remark this answers is not in the document]
Comment (adiramani) [reply endgame_postprocessing/post_processing/single_file_post_processing.py:107]: That is a fair point. I'm a bit used to what we did at my previous work where customers had legacy implementations of our package that we had to constantly try to keep compatible with future versions 😄. I can remove this.

File: endgame_postprocessing/model_wrappers/constants.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 
+THRESHOLDS = [0.50, 0.75, 0.85, 0.90, 0.95]
+
 # Country Level Params
 COUNTRY_SUMMARY_COLUMNS = [
     "prevalence",
     "year_of_threshold_prevalence_avg",
-    "year_of_pct_runs_under_threshold",
-]
+] + [f"year_of_pct_runs_under_threshold_{int(thresh * 100)}" for thresh in THRESHOLDS]
```
[the remark this answers is not in the document]
Comment (adiramani) [reply endgame_postprocessing/model_wrappers/constants.py:9]: What about `f"year_of_{int(thresh * 100)}pct_runs_under_threshold"`?

File: endgame_postprocessing/post_processing/aggregation.py [hunk 4]
```diff
@@ -160,30 +165,52 @@ def country_lvl_aggregate(
...
         if measure_val in threshold_cols_rename
     ]
-    return pd.concat([general_summary, summarize_threshold], axis=0, ignore_index=True)
+
+    summarize_threshold_counts = (
+        iu_lvl_data[
+            iu_lvl_data[MEASURE_COLUMN_NAME].isin(threshold_summary_measure_names)
+        ]
+        .groupby(threshold_groupby_cols)
+        .agg({"mean": [("mean", _calc_sum_not_na)]})
```
[the remark this answers is not in the document]
Comment (thk123) [reply endgame_postprocessing/post_processing/aggregation.py:199]: Thanks for the explanation about the tuple, of course, I think I get tripped up by the different measures being different rows. What you've done makes sense.

**Comment by thk123** [comment]
Clarified on a call, that for countries we now have

Percentage of IUs that reached threshold in X percent of runs. 
Number of IUs that reached threshold in X percent of runs. 

I will double check these aren't showing up in LF and then send you files if not.

**Pull request merged**